student.rb

```diff
@@ -5,7 +5,14 @@ def initialize(_clasroom, age, name = 'unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  attr_accessor :owner
+
   def play_hooky
     "¯\(ツ)/¯"
   end
+
+  def classroom=(classroom)
+    @classrom = classroom
```

Comment:
- [ ] Kindly fix your syntax error here. `@classrom` instance variable is not defined in your class. I think you meant `@classroom` (without two o's)